### DISCUSSION TITLE
请问：如何使用SVP4补帧？

### DISCUSSION THREAD

**Discussion by Nielson-cn** [422658476/MPV-EASY-Player#83] [Q&A【问答】]
按SVP4的说明[https://www.svp-team.com/wiki/SVP:mpv](url)，要在mpv.conf中增加一句“input-ipc-server=mpvpipe”。

但是，
如果加在 \data\portable-data\mpv.conf里，则没有效果。
如果加在 \data\mpv-easy-data\rjno1.conf里，则MPV Easy Player无法播放视频文件——打开文件、拖入文件、点播放键都没有反应。

请问应该如何解决？谢谢！

**Comment by Vulcankta** [comment]
我也有相同疑問

**Comment by 422658476** [comment]
请在mpv中进行这个操作。
mpv-easy需要使用input-ipc-server参数和mpv通信，会和这个冲突，也就是你不能在mpv-easy中使用这个。
